src/api/customer/customer-controller.ts

```diff
@@ -12,9 +12,36 @@ const createCustomer = (): CustomerCreationRequestHandler[] => {
         persistCustomer,
         createCustomerSerializer,
         (req, res) => { res.status(CREATED).json(res.locals.customerToRespond) }
-    ]
-}
+    ];
+};
+
+const getCustomersAll = (): GetCustomersAllRequestHandler [] => {
+    return [
+        getCustomersAllValidator(),
+        paginationSerializer,
+        getCustomerAllBusinees,
+        getAllCustomersSerializer,
+        (req, res) => {
+            res.status(OK).json(res.locals.customerToRespond);
+        },
+    ];
+};
+
+const getCutomerById = (): GetCustomerByIdRequestHandler [] => {
+    return [
+        getCustomerByIdValidator(),
+        getCustomerByIdBusinees,
+        getCustomerByidSerializer,
+        (req, res) => {
+            if (!res.locals.customerToRespond) {
+                res.status(NOT_FOUND).json(res.locals.customerToRespond);
+            } else {
+                res.status(OK).json(res.locals.customerToRespond);
+            }
```

Comment:
Para melhorar a legibilidade do código, considere remover a negação `!`, e inverter a ordem do `if`.

```suggestion
            if (res.locals.customerToRespond) {
                res.status(OK).json(res.locals.customerToRespond);
            } else {
                res.status(NOT_FOUND).json(res.locals.customerToRespond);
            }
```